recipes/nsls2-collection/meta.yaml

```diff
@@ -0,0 +1,122 @@
+{% set name = "nsls2-collection" %}
+{% set year = "2021" %}
+{% set cycle = "2" %}
+{% set version = "0" %}
+{% set micro = "" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ year }}C{{ cycle }}.{{ version }}{{ micro }}
+
+build:
+  skip: true  # [py<37 or not linux]
+  number: 0
+
+requirements:
+  host:
+    - python
+  run:
+    - nsls2-analysis ==2021C2.0=*_0
+    - python
+    - bloptools
+    - bluesky-darkframes
+    - caproto
+    - happi
+    - nslsii >=0.1.3
+    - pexpect
+    - pyepics >=3.4.2
+    - pyolog >=4.4.0
+    - pyserial
+    - python-confluent-kafka  # [not win]
+    - redis-py
+    - simple-pid
+    - slack-sdk
+    # Beamline-specific packages
+    - hklpy  # [linux]
+    - hxnfly
+    - ppmac
+    - xpdacq
+    # Debugging tools:
+    - hunter
+    - logging_tree
+    # Profiling tools:
+    - line_profiler
+    - pyinstrument
+
+test:
+  imports:
+    - area_detector_handlers
+    - bluesky
+    - caproto
+    - dask
+    - databroker
+    - event_model
+    - lmfit
+    - matplotlib
+    - napari
+    - nslsii
+    - numpy
+    - ophyd
+    - oscars
+    - peakutils
+    - photutils
+    - pymongo
+    - pyxrf
+    - redis
+    - simple_pid
+    - skbeam
+    - srwlpy  # [linux]
```

Comment:
This import test was intended to be run against the `srw` package coming from https://github.com/nsls-ii-forge/srw-feedstock/, which we don't package in conda-forge, where `srwpy` (see below) should be the right one.
```suggestion
```